services/data_plane_manager/src/test/java/com/futurewei/alcor/dataplane/utils/GoalStateManagerTest.java

```diff
@@ -41,310 +41,121 @@ public boolean shouldSkipClass(Class<?> clazz) {
   @AfterEach
   void tearDown() {}
 
-  @Test
-  public void scenarioC() {
+ @Test
```

Comment:
This file become a humongous test file. Consider to break down into one main file with only the test logic and many payload file. This way, you can find it relatively easy to debug and add a new test case. @haboy52581